content/200-orm/200-prisma-client/500-deployment/301-edge/100-overview.mdx

```diff
@@ -0,0 +1,58 @@
+| Vercel Edge Functions   | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Vercel Edge Middleware  | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Cloudflare Workers      | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Cloudflare Pages        | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Deno Deploy             | [Not yet](https://github.com/prisma/prisma/issues/2452) | ✅                               |
+| Netlify Edge Functions  | Not yet                                                 | ✅                               |
+| Supabase Edge Functions | Not yet                                                 | ✅                               |
+
+Deploying edge functions that use Prisma ORM on Cloudflare and Vercel is currently in [Preview](/orm/more/releases#preview).
+
+## Edge-compatibility of database drivers
+
+### Why are there limitations around database drivers in edge functions?
+
+Edge functions typically don't use the standard Node.js runtime. For example, Vercel Edge Functions and Cloudflare Workers are running code in [V8 isolates](https://v8docs.nodesource.com/node-0.8/d5/dda/classv8_1_1_isolate.html). Deno Deploy is using the [Deno](https://deno.com/) JavaScript runtime. As a consequence, these edge functions only have access to a small subset of the standard Node.js APIs and also have constrained computing resources (CPU and memory).
+
+In particular, the constraint of not being able to freely open TCP connections makes it difficult to talk to a traditional database from an edge function. While Cloudflare has introduced a [`connect()`](https://developers.cloudflare.com/workers/runtime-apis/tcp-sockets/) API that enables limited TCP connections, this still only enables database access using specific database drivers that are compatible with that API.
+
+> **Note**: [Prisma Accelerate](/accelerate) enables you to access _any_ database from _any_ edge function provider. No edge-compatible driver is necessary.
+
+### Which database drivers are edge-compatible?
+
+Here is an overview of the different database drivers and their compatibility with different edge function offerings:
+
+- [Neon Serverless](https://neon.tech/docs/serverless/serverless-driver) uses HTTP to access the database
+- [PlanetScale Serverless](https://planetscale.com/docs/tutorials/planetscale-serverless-driver) uses HTTP to access the database
+- [`node-postgres`](https://node-postgres.com/) (`pg`) uses Cloudflare's `connect()` (TCP) to access the database
+- [`@libsql/client`](https://github.com/tursodatabase/libsql-client-ts) is used to accessed Turso databases
+
+> **Note**: There's [also work being done](https://github.com/sidorares/node-mysql2/pull/2289) on the `node-mysql2` driver which will enable to access traditional MySQL databases from edge functions as well.
```

Comment:
Probably should mention that this will only make it compatible with Cloudflare via `connect()` as well, similar to `pg`.

Reply:
good point. I also added another sentence after each bullet to indicate which edge function offerings these are compatible with.